temporal/api/cloud/nexus/v1/message.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package temporal.api.cloud.nexus.v1;
+
+option go_package = "go.temporal.io/api/cloud/nexus/v1;nexus";
+
+import "google/protobuf/timestamp.proto";
+
+message EndpointSpec {
+    // The name of the endpoint. Must be unique within an account.
+    // The name must match `[a-zA-Z_][a-zA-Z0-9_]*`.
+    // This field is mutable.
+    string name = 1;
+
+    // Indicates where the endpoint should forward received nexus requests to. 
+    EndpointTargetSpec endpoint_target_spec = 2;
```

Comment:
```suggestion
    EndpointTargetSpec target_spec = 2;
```

Inconsistent in whether you choose to prefix fields in this message with "endpoint_"

Reply:
removed endpoint_ suffix